apps/condo/pages/tour/guide.tsx

```diff
@@ -86,11 +86,11 @@ const AboutAppBlock = () => {
     const InMoreDetailMessage = intl.formatMessage({ id: 'InMoreDetail' })
     const NextMessage = intl.formatMessage({ id: 'Next' })
     const CloseMessage = intl.formatMessage({ id: 'Close' })
-    const ModalCardLinkMessage = intl.formatMessage({ id: 'tour.guide.aboutApp.modal.card.link' })
 
     const { breakpoints } = useLayoutContext()
     const locale = useMemo(() => get(intl, 'locale'), [intl])
 
+    const modalCardExamples = get(guideModalCardExamples, [locale, 'types'], {})
```

Comment:
Please describe the structure of your config or add example. 

Each locale should have its own text/links/pictures.
It shouldn't be that in the English interface I will have a picture with Russian letters

Reply:
This changes only for ru locale. In en locale nothing should happen

Reply:
> This changes only for ru locale. In en locale nothing should happen

It's better to support any locale.
If tomorrow they tell you we want hints in English or any other language, then for this you only need to update envs.

And we are also open source :)